OxfordBodleian/Juel-Jensen/BDLaethf32.xml

```diff
@@ -124,51 +154,129 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">10</dim>
                                         <dim type="left">8</dim>
                                     </dimensions>
+                                    <note> The characters per line are 5-9</note>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.</ab>
                                 </layout>
                             </layoutDesc>
-                           
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <desc>
+                                    Mediocre handwriting. Slant and spacing of the characters vary throughout the manuscript.
+                                </desc>
+                                <seg type="rubrication">
+                                    Holy names; some lines on the incipit page, alternating with black lines; few lines on <locus target="#62v"/>;
+                                    the title of the <foreign xml:lang="gez">Ṣalota fattǝto</foreign> on <locus target="#64v"/>;
+                                    liturgical directives in abbreviation; few letters, alternating with black lines, of single words or groups of words on <ref target="#63v #71v #72v"/>;                                    
+                                    elements of the punctuation signs and numerals.
+                                </seg>
+                                <list type="abbreviations">
+                                    <item>
+                                        <abbr>ይዲ</abbr> for <expan>ይቤ፡ ዲያቆን፡</expan> (e.g. <locus target="#15r #17v #18v"/>)
```

Comment:
I think it stands for "ይበል፡ ዲያቆን፡" (except in the AksColl, which writes "ወይብል፡"). The same for ካህን፡ and ሕዝብ፡.

Reply:
Thanks Jonas for this clarification!